src/core/markdown.js

```diff
@@ -200,6 +200,11 @@ function normalizePadding(text) {
         }
         return chop;
       }, chop);
+    for (const pre of doc.querySelectorAll("pre")) {
```

Comment:
Seems a little redundant to redo the querySelector here, as we already have all the `pre` elements up on line 191. I wonder if we can combine these (either by making a variable or by doing this as part of the chain above)?